tests/kernel/wave/attention/vanilla_attention_test.py

```diff
@@ -1083,6 +1083,7 @@ def testAttentionBSHD_Prefetch_MultiBuffer(
         benchmark_batch_size=10,
         benchmark_repetitions=3,
         benchmark_results_file=perf_filename_tk,
+        multi_buffer_count=2,  # TODO: Hack as schedule reordering doesn't respect lifetimes computed during the main scheduling algorithm.
```

Comment:
Does that mean that if we specify our own schedule we need to manually set the multi_buffer_count?

Reply:
`override_schedule` is consumed before the loop reconstruction so it should work fine without specifying multibuffer count manually.